packages/model-viewer/src/three-components/gltf-instance/correlated-scene-graph.ts

```diff
@@ -1,8 +1,10 @@
-import {Group, Material, Mesh, Object3D, Texture} from 'three';
+import {Group, Material, Mesh, MeshStandardMaterial, Object3D, Texture} from 'three';
 import {GLTF as ThreeGLTF, GLTFReference} from 'three/examples/jsm/loaders/GLTFLoader.js';
 
 import {GLTF, GLTFElement} from '../../three-components/gltf-instance/gltf-2.0.js';
 
+
+
```

Comment:
With your tests failing iOS 13 it looks like you may have forgotten to pull master before you branched this? Anyway, a pull now should be a quick fix.

Reply:
Done.